packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -38,6 +38,29 @@ const Duration _kExpand = Duration(milliseconds: 200);
 /// ** See code in examples/api/lib/material/expansion_tile/expansion_tile.0.dart **
 /// {@end-tool}
 ///
+/// ## Troubleshooting
+///
+/// ### Why do I need to provide unique PageStorageKey(s) to scrollable(s) inside [ExpansionTile]
+/// when providing a PageStorageKey [ExpansionTile]?
+///
+/// When providing a unique PageStorageKey [ExpansionTile] to save and restore its expanded state,
```

Comment:
This sentence is super long and hard to parse. Can you split this up into multiple sentences to make it easier to read?